client/components/NoEvents.jsx

```diff
@@ -0,0 +1,18 @@
+import React from 'react';
+import { Link } from 'react-router-dom';
+
+
+const NoEvents = () => (
+  <div className="no-events">
+    <div className="no-events__container">
+      <h1 className="no-events__status-message">
+        There are no upcoming events at the moment
```

Comment:
So I figured we were using a single day view and that a better message will be: `There are no new events for today. You can update your` [interests]() `so that you are notified when new events come up.` Apologies for not having this documented before hand.